src/Playwright/Helpers/Paths.cs

```diff
@@ -33,31 +33,38 @@ internal static class Paths
     {
         internal static string GetExecutablePath()
         {
-            DirectoryInfo assemblyDirectory = new(AppContext.BaseDirectory);
-            if (!assemblyDirectory.Exists || !File.Exists(Path.Combine(assemblyDirectory.FullName, "Microsoft.Playwright.dll")))
+            /*
+             * Scenarios this has to cover:
+             * 1) The simple one, where the .playwright folder is next to the BaseDirectory, which is almost always
+             * 2) Scenarios where we have to rely on the assembly location of Microsoft.Playwright.dll, but can't be
+             *      sure that's always correct, because it can be empty for some cases (hence 1. is first)
+             * 3) Scenarios where we get loaded from NuGet and **NOT** copied, which is rare, but implies we're loaded
```

Comment:
Could you include the file tree snippet in the comment to make it clear why drivers are not next to .dll and we need to climb the filesystem tree? I.e. why (2) does not cover (3). Same would be interesting for (1), i.e. when (2) does not cover (1).